doc/deployment-on-production-env.md

```diff
@@ -0,0 +1,43 @@
+# Deployment on Production Environment
+
+In this guide we will cover installation of pre-requisites on your 
+production cluster.
+
+# Requirements
+
+## A Cluster
+
+This guide requires you to have your own Kubernetes/OpenShift cluster.
+
+## Install Supported Multi-Network CNI
+
+Install any supported [Network Plumbing Working Group de-facto standard](https://github.com/k8snetworkplumbingwg/multi-net-spec) implementation.
+For example - [Multus](https://github.com/intel/multus-cni). To deploy multus on a kubernetes cluster with flannel cni:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/hack/multus/kubernetes-multus.yaml
+kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/hack/multus/multus.yaml
+```
+
+## Install CNI plugins
+
+* [CNI plugins](https://github.com/containernetworking/plugins) must be installed in the cluster, to allow collaboration with 
+CNIs.
+To deploy CNI plugins on a kubernetes cluster:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/hack/cni-plugins/cni-plugins.yaml
+```
+
+# Deploy Kubemacpool Manager
+
+Download the project release manifest yaml and modify its mac range to avoid collisions with nearby clusters.
+Finally, deploy the project by applying its yaml.
+
+```bash
+wget https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/config/release/kubemacpool.yaml
```

Comment:
This makes me think - could we just refer to CNAO as the operator that should be used for production? We would avoid duplicity. On the other hand, some users may welcome simple, straight to the point guide on how to use this standalone. Leaving it to you

Reply:
I think deploying via CNAO would be best. it will avoid any future version breakages. we know that if they use CNAO version X then they also use multus version y and kmp z that were tested together..
plus we get a free commercial for CNAO :)